src/main/scala/latis/util/hapi/parameter.scala

```diff
@@ -25,21 +25,21 @@ final case class ScalarParameter(
 final case class VectorParameter(
   name: String,
   typeName: String,
-  units: String,
+  units: Option[String],
   length: Option[Int],
   fill: Option[String],
-  size: Int
+  size: List[Int]
```

Comment:
I think this (and `size` and `bin` in `ArrayParameter`) should be `NonEmptyList`s.